src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let resultSum = 0;
+
+  const funAdder = (a) => {
+    if (a === undefined) {
```

Comment:
It's a good practice to explicitly check for `undefined` when you expect a function could be called without arguments. Well done on this check.